config/db_setup.py

```diff
@@ -1,3 +1,3 @@
 import postgresql
 
-db = postgresql.open("pq://username:password@localhost/databasename")
\ No newline at end of file
+db = postgresql.open("pq://postgres:mypwd@localhost/mydatabase")
```

Comment:
I think these specs are specific to your machine. Which setting allows all of us to access Postgres without a hassle?

Reply:
If we are still planning to use SQLAlchemy (the ORM), the database will be pointed to when creating an 'engine'. I'm looking into it.